tests/functional/pages/multiple-survey.page.js

```diff
@@ -1,9 +0,0 @@
-class MultipleSurveys {
-
-  hasMultipleSurveyError() {
-    return browser.isExisting('[data-qa="multiple-survey-error"]')
-  }
-
-}
-
-export default new MultipleSurveys()
```

Comment:
Do we still get the multiple surveys error on GET requests to 'old' surveys?

Reply:
Yes and I've changed the behaviour for the CSRF failing to display the same message rather the "stale session" message now. Means that as far as a user is concerned nothing has changed.